src/uwtools/scheduler.py

```diff
@@ -34,15 +32,41 @@ class JobScheduler(collections.UserDict):
 
     _map = {}
     prefix = ""
+    key_value_separator = "="
 
     def __getattr__(self, name):
         if name in self:
             return self[name]
+        raise AttributeError(name)
 
     @property
-    @abstractmethod
-    def job_card(self) -> JobCard:
-        raise NotImplementedError
+    def job_card(self):
+        """returns the job card to be fed to external scheduler"""
+        ignored_keys = ["scheduler"]
+
```

Comment:
Is it appropriate to pop scheduler from self._map and set it as an attribute in __init__ (or just pop it where ever it might be used)? That would reduce the logic here a bit.

Reply:
That makes sense.